test/cpp/api/tensor.cpp

```diff
@@ -915,6 +915,12 @@ TEST(TensorTest, Arange) {
     auto x = torch::arange(0, 5);
     ASSERT_EQ(x.dtype(), torch::kLong);
   }
+  {
```

Comment:
Ho I don't know how these tests work, could you also have a test on the python side?

Reply:
Sorry for the mistake. I have made changes.